### PR TITLE
#317 TkReturn and RsReturn

### PR DESCRIPTION
**Pull request by komaz**
Fix for issue #317. Added `TkReturn` and `RsReturn` classes from netbout (with proper change of packages, copyrights and versions), created test cases.

### CODE REVIEW

**Comment by davvd** [comment]
@komaz I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by davvd** [comment]
@caarlos0 it's yours, please review

File: src/main/java/org/takes/facets/ret/RsReturn.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                "Path=/",
+                String.format(
+                    Locale.ENGLISH,
+                    "Expires=%1$ta, %1$td %1$tb %1$tY %1$tT GMT",
+                    new Date(
+                        System.currentTimeMillis()
+                            + TimeUnit.HOURS.toMillis(1L)
+                    )
+            )
+        )
```
Comment (caarlos0) [inline src/main/java/org/takes/facets/ret/RsReturn.java]: @komaz indentation seem wrong here...

**Comment by caarlos0** [comment]
@komaz just one comment above..

**Comment by caarlos0** [comment]
@komaz also, be aware of v

**Comment by komaz** [comment]
@caarlos0 looks like it is some bug in checkstyle, fixed indentation and added checkstyle ignore. Also found missing public modifier, fixed.

**Comment by caarlos0** [comment]
@komaz thanks!

**Comment by caarlos0** [comment]
@rultor good to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@ypshenychka please, review this ticket for compliance with our [QA rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor deploy pls

**Comment by ypshenychka** [comment]
@caarlos0 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one cosmetic issue was found during code review. Try to find at least three problems while future reviews.

Please confirm that everything is clear and you have no questions regarding this approach.

**Comment by caarlos0** [comment]
@ypshenychka  sure, I'll try

**Comment by ypshenychka** [comment]
@caarlos0 Thank you!

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks, got it. everybody, please try to make it good next time

**Comment by davvd** [comment]
@caarlos0 just added 10 mins to @ypshenychka (for QA), payment ID is `58626378`; done, I added **19 mins** in payment `AP-9VP0241221273962X`, 47 hours and 53 mins spent to complete this; review comments (c=4) added as a bonus; added +19 to your rating, now it is equal to [+2032](http://www.netbout.com/b/35095?open=rating)

**Pull request merged**